engines/private/funcs.cpp

```diff
@@ -731,80 +747,79 @@ static void fTimer(ArgArray args) {
 const FuncTable funcTable[] = {
 
 	// Control flow
-	{ fChgMode,		 "ChgMode"},
-	{ fResume,		  "Resume"},
-	{ fgoto,			"goto"},
-	{ fTimer,		   "Timer"},
+	{fChgMode, "ChgMode"},
+	{fResume, "Resume"},
+	{fgoto, "goto"},
```

Comment:
this is not consistently named. `fGoto`?

Reply:
`goto` is in lowercase in the game scripts, either as keyword `goto X` and as a function `goto(X)`. The other ones are functions (e.g. `Resume`, `ChgMode`, etc) and always start with uppercase. Anyway, I can rename it you want.